audio.js

```diff
@@ -0,0 +1,10 @@
+function PlaySound(soundobj) {
```

Comment:
@andrewnester Thank you for your note. It has happen because of high .Net concentration in my blood.